README.md

```diff
@@ -51,7 +52,19 @@ pnpm hardhat:deploy
 
 This command deploys a test smart contract to the local network. The contract is located in `packages/hardhat/contracts` and can be modified to suit your needs. The `pnpm hardhat:deploy` command uses the deploy script located in `packages/hardhat/deploy` to deploy the contract to the network. You can also customize the deploy script.
 
-4. On a third terminal, start your NextJS app:
+4. Ensure docker is running and then on a third terminal, start-up indexer:
+
+```
+pnpm envio:dev
```

Comment:
Need to add `pnpm envio:codegen` as init is never run when this is cloned

https://github.com/Float-Capital/indexer/issues/974